server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreRestoreIT.java

```diff
@@ -396,6 +400,17 @@ public void testRateLimitedRemoteDownloads() throws Exception {
             shardCount,
             buildRemoteStoreNodeAttributes(REPOSITORY_NAME, randomRepoPath(), REPOSITORY_2_NAME, randomRepoPath(), true)
```

Comment:
can we set this to not apply `max_remote_download_bytes_per_sec` here and then apply it in inplace . That way we will be sure that it gets really applied .

Reply:
Updated